gain_value: 14.0
gain_auto: true
gamma_selector: 0
gamma_value: 0.7
exposure_auto: true
exposure_value: 20.0
exposure_max: 20.0
exposure_min: 20.0
white_auto: true
white_selector: 0